docs/docsite/rst/network/user_guide/cli_parsing.rst

```diff
@@ -0,0 +1,728 @@
+     getval: 'admin state is (?P<admin_state>\S+),'
+     result:
+       "{{ name }}":
+         name: "{{ name }}"
+         state:
+           admin: "{{ admin_state }}"
+
+   - example: "  Hardware: Ethernet, address: 5254.005a.f8b5 (bia 5254.005a.f8b5)"
+     getval: '\s+Hardware: (?P<hardware>.*), address: (?P<mac>\S+)'
+     result:
+       "{{ name }}":
+         hardware: "{{ hardware }}"
+         mac_address: "{{ mac }}"
+
+
+This native parser template is structured as a list of parsers, each
+containing the following key-value pairs:
+
+-  ``example`` - An example line of the text line to be parsed
+-  ``getval`` - A regular expression using named capture groups to store the extracted data
+-  ``result`` - A data tree, populated as a template, from the parsed data
+-  ``shared`` - (optional) The shared key makes the parsed values available to the rest of the parser entries until matched again.
+
+The following example task uses ``cli_parse`` with the native parser and the example template above to parse the ``show interface`` command from a Cisco NXOS device:
+
+.. code-block:: yaml
+
+   - name: "Run command and parse with native"
+     ansible.netcommon.cli_parse:
+       command: show interface
+         parser:
+           name: ansible.netcommon.native
+       set_fact: interfaces
+
+Taking a deeper dive into this task:
+
+- The ``command`` option provides the command you want to run on the device or host. Alternately, you can provide text from a previous command with the ``text`` option instead.
+- The ``parser`` option provides information specific to the parser engine.
+- The ``name`` suboption provides the fully qualified collection name (FQCN) of the parsing engine (``ansible.netcommon.native``).
+- The ``cli_parse`` module, by default, looks for the template in the templates directory as ``{{ short_os }}_{{ command }}.yaml``.
+
+  - The ``short_os`` in the template filename is derived from either the host ``ansible_network_os`` or ``ansible_distribution``.
+  - Spaces in the network or host command are replace with ``_`` in the ``command`` portion of the template filename. In this example, the ``show interfaces`` network CLI command becomes ``show_interfaces`` in the filename.
+
+.. note::
+
+	``ansible.netcommon.native`` parsing engine is fully supported with a Red Hat Ansible Automation Platform subscription.
+
+Lastly in this task, the ``set_fact`` option sets the following ``interfaces`` fact for the device based on the now-structured data returned from ``cli_parse``:
+
+.. code-block:: yaml
+
+   Ethernet1/1:
+       hardware: 100/1000/10000 Ethernet
+       mac_address: 5254.005a.f8bd
+       name: Ethernet1/1
+       state:
+       admin: up
+       operating: up
+   Ethernet1/10:
+       hardware: 100/1000/10000 Ethernet
+       mac_address: 5254.005a.f8c6
+   <...>
+
+
+Linux example
+^^^^^^^^^^^^^
+
+You can also use the native parser to run commands and parse output from Linux
+hosts.
```

Comment:
There are a bunch of places like this where short paragraphs are broken into multiple lines. This will make translation more difficult. Can we consolidate a bit?

Reply:
the joy of automated md to rst conversion. I think I caught the remaining ones now